src/store/metadata/stub/content/datasets.json

```diff
@@ -276,8 +276,53 @@
           "modified": "2023-11-02T14:41:11Z"
         }
       ]
+    },
+    {
+      "@id": "https://staging.idpd.uk/datasets/schools-and-nursery-schools-rated",
+      "@type": "dcat:DatasetSeries",
+      "identifier": "schools-and-nursery-schools-rated",
+      "title": "Schools and nursery schools rated good or outstanding",
+      "summary": "Percentage of state-funded provided for schools that are rated",
```

Comment:
Change this to 'Percentage of state-funded schools that are rated good or outstanding'